File: problemas/strings/1632/1632.py
def main():
    T = int(input())
    letras = {'a', 'e', 'i', 'o', 's', 'A', 'E', 'I', 'O', 'S'}

    for _ in range(T):
        count = 1
        senha = str(input().strip())

        for letra in senha:
            if letra in letras:
                count *= 3
            else:
                count *= 2
        print(count)

if __name__ == "__main__":
    main()

File: problemas/strings/1632/1632.js
process.stdin.resume();
process.stdin.setEncoding('utf8');

let input = "";
process.stdin.on('data', (chunk) => {
    input += chunk;
});

process.stdin.on('end', () => {
    const entrada = input.trim().split('\n');
    const T = parseInt(entrada.shift(), 10);
    const especiais = ['a', 'e', 'i', 'o', 's', 'A', 'E', 'I', 'O', 'S'];

    for (let i = 0; i < T; ++i) {
        let count = 1;
        const senha = entrada.shift().trim();

        for (const char of senha) {
            if (especiais.includes(char)) {
                count *= 3
            } else {
                count *= 2
            }
        }

        console.log(count);
    }
});
